### ISSUE TITLE
The pre-trained detector

### ISSUE THREAD

**Issue by 2023zy** [Huy0110/co-separation-with-cross-modal-loss#1]
Hi, Huy
The author of the paper shared the pre-trained detector at Google Drive, but I couldn't find it when accessing the link. It seems that there is nothing in the folder now.  Would it be convenient for you to provide it?
thanks
